docs/docs/segment-aws.md

```diff
@@ -0,0 +1,38 @@
+---
+id: aws
+title: AWS Context
+sidebar_label: AWS
+---
+
+## What
+
+Display the currently active AWS profile and region.
+
+## Sample Configuration
+
+```json
+{
+  "type": "aws",
+  "style": "powerline",
+  "powerline_symbol": "\uE0B0",
+  "foreground": "#ffffff",
+  "background": "#FFA400",
+  "properties": {
+    "prefix": " \uE7AD ",
+    "template": "{{.Profile}}{{if .Region}}@{{.Region}}{{end}}"
+  }
+}
+```
+
+## Properties
+
+- template: `string` - A go [text/template][go-text-template] template extended with [sprig][sprig] utilizing the
+properties below. Defaults to `{{.Context}}{{if .Namespace}} :: {{.Namespace}}{{end}}`
```

Comment:
@JanDeDobbeleer the default looks incorrect here.

Reply:
You're right 🤦🏻‍♂️